monitoring/timedrift_test.go

```diff
@@ -126,7 +140,7 @@ func (s *TimeDriftSuite) TestTimeDriftChecker(c *check.C) {
 				Clock:      s.clock,
 			},
 			FieldLogger: logrus.WithField(trace.Component, "test"),
-			clients:     newClientsCache(test.times),
+			clients:     newClientsCache(test.times, test.slow),
```

Comment:
Why not just use an expired context? You can create one in the SetUpSuite or as a package global:
```go
s.ctx, cancel = context.WithCancel(context.Background())
cancel()
```
and then just use it in the test instead of `slow` - this will require minimum changes.